Assets/SEE/GameObjects/SEESpline.cs

```diff
@@ -411,17 +469,67 @@ public Vector3[] GenerateVertices()
         /// </summary>
         private void UpdateColor()
         {
-            if (gameObject.TryGetComponent(out LineRenderer lr))
+            if (lineRenderer != null)
             {
-                lr.startColor = gradientColors.start;
-                lr.endColor = gradientColors.end;
+                lineRenderer.startColor = gradientColors.start;
+                lineRenderer.endColor = gradientColors.end;
             }
             else if (meshRenderer != null)
             {
                 UpdateMaterial();
             }
         }
 
+        private void UpdateCollider()
+        {
+            if (gameObject.TryGetComponent(out Collider collider))
+            {
+                collider.enabled = IsSelectable;
+                return;
+            }
+
+            if (IsSelectable)
+            {
+                if (!gameObject.TryGetComponent(out MeshFilter filter))
+                {
+                    Debug.LogWarning("Trying to update selectability without generating a mesh first!");
+                    return;
+                }
+                Mesh mesh = filter.sharedMesh;
+
+                MeshCollider meshCollider = gameObject.AddOrGetComponent<MeshCollider>();
+                // IMPORTANT: Null the shared mesh of the collider before assigning the updated mesh.
+                // https://forum.unity.com/threads/how-to-update-a-mesh-collider.32467/
+                meshCollider.sharedMesh = null; // Do we still need this workaround?
```

Comment:
Feel free to test it :)

Reply:
You mean removing the workaround?

Reply:
In theory, one should just move a node and see if the edge collider still matches the visual edge.